packages/apps/transfer/src/services/transfer/coin-transfer.ts

```diff
@@ -0,0 +1,68 @@
+import { PactCommand } from '@kadena/client';
+import { sign } from '@kadena/cryptography-utils';
+
+import { onlyKey } from '../utils/utils';
+
+const NETWORK_ID = 'testnet04';
+const chainId = '1';
+export const API_HOST = `https://api.testnet.chainweb.com/chainweb/0.0/${NETWORK_ID}/chain/${chainId}/pact`;
+const gasLimit = 2300;
+const gasPrice = 0.00001;
+const ttl = 28800;
+
+export interface TransferResult {
+  requestKey?: string;
+  status?: string;
+}
+
+export async function makeTransferCreate(
```

Comment:
The name is a bit vague still. It's sounds like it's preparing the transfer object for you, but it's also sending coins already it seems